lib/dest/write-contents/index.js

```diff
@@ -38,26 +40,13 @@ function writeContents(opt) {
     // This is invoked by the various writeXxx modules when they've finished
     // writing the contents.
     function onWritten(writeErr) {
-      if (isErrorFatal(writeErr)) {
+      if (fo.isOverwriteError(writeErr, file.flag)) {
```

Comment:
But that also seems slightly off, because the function returns true if it _is_ an overwrite error (`EEXIST`) but we want to ignore it (because `overwrite` was false)?